lib/web_ui/lib/src/ui/canvas.dart

```diff
@@ -302,7 +302,7 @@ abstract class Canvas {
   ///    for subsequent commands.
   ///  * [BlendMode], which discusses the use of [Paint.blendMode] with
   ///    [saveLayer].
-  void saveLayer(Rect/*?*/ bounds, Paint/*!*/ paint);
+  void saveLayer(Rect? bounds, Paint paint);
```

Comment:
How are you validating that these APIs migrated to the same types as the other dart:ui?

Reply:
Fixed `api_confrom_test.dart`, but I'm not sure how to validate parameters. Can't find the right fields in the AST nodes :thinking: (maybe just tired at the end of the week)